src/components/bottom-widget.tsx

```diff
@@ -18,18 +18,25 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-import React, {useCallback, forwardRef, useMemo} from 'react';
+import React, {useCallback, forwardRef, useMemo, ReactNode} from 'react';
 import styled from 'styled-components';
 import TimeWidgetFactory from './filters/time-widget';
 import AnimationControlFactory from './common/animation-control/animation-control';
 import AnimationControllerFactory from './common/animation-control/animation-controller';
 import {ANIMATION_WINDOW, DIMENSIONS, FILTER_TYPES} from 'constants/default-settings';
 import {getIntervalBins} from 'utils/filter-utils';
 import {media} from 'styles/media-breakpoints';
+import {AnimationConfig, TimeRangeFilter} from 'reducers';
+import {bottomWidgetSelector} from './kepler-gl';
 
 const maxWidth = 1080;
 
-const BottomWidgetContainer = styled.div`
+interface BottomWidgetContainer {
```

Comment:
type should have `Props` in its name, so it won't be conversion when exporting both the component and the type of its prop
```js
interface BottomWidgetContainerProps {
  hasPadding?: boolean;
  width: number
}
```